examples/mobile/HomeApp/js/app.js

```diff
@@ -6,60 +6,100 @@ import Storage from "./clipping-storage.js";
 	var tau = window.tau,
 
 		storage = new Storage(),
-		webClipList = [],
-		activeWebClipList = storage.readAllFromStorage(Storage.elements.ACTIVEWEBCLIPS) || [
-			"webclip/tv-remote-control",
-			"webclip/restaurant",
-			"webclip/weather"
-		];
+		appsList = [];
 
 	function changeTheme(event) {
 		tau.theme.setTheme(event.target.value);
 	}
 
 	function loadWebClipList() {
-		const requestURL = "api/webcliplist";
+		const requestURL = "api/appslist";
 
 		fetch(requestURL)
 			.then((response) => response.json())
 			.then((data) => {
-				webClipList = data;
+				appsList = data;
+				appsList.forEach((app) => {
+					app.webClipsList.forEach((webclip) => {
+						webclip.isSelected = false;
+					});
+				});
 			})
 			.catch(() => {
-				// use default webclip list is sth wrong
-				webClipList =
-					[
-						"webclip/tv-remote-control",
-						"webclip/restaurant",
-						"webclip/weather",
-						"webclip/apps-on-tv",
-						"webclip/latest-news",
-						"webclip/now-on-tv"
-					];
+				// use default webclip list if sth wrong
+				appsList = [
```

Comment:
Please, move this object to some object variable for module like defaultList.

Reply:
The structure of this object seems appropriate to me. We can change it during the project development.

Reply:
done